SkietbaanBE/SkietbaanBE/Helper/NotificationMessages.cs

```diff
@@ -75,6 +77,48 @@ public void GroupNotification(ModelsContext _context, Group group, User user)
                 message = e.Message;
             }
         }
+
+        public void LOS(string token)
+        {
+            var user = _context.Users.SingleOrDefault(x => x.Token == token);
+            var notification = new Notifications
+            {
+                User = user,
+                IsRead = false,
+                TypeOfNotification = "Document",
```

Comment:
what are the other types of notifications? because if there are more than 1 (which I can see is  *Document*) then I suggest you use Enum.